Code/JavaWrappers/csharp_wrapper/CMakeLists.txt

```diff
@@ -31,7 +31,9 @@ endif(APPLE)
 
 if(CMAKE_SIZEOF_VOID_P EQUAL 4)
   SET(CMAKE_SWIG_FLAGS -namespace "GraphMolWrap")
+  SET(PLATFORM "x86")
```

Comment:
Curiosity: I wonder if there are any circumstances where we may be building the C# wrappers on something like an ARM CPU?

Reply:
That might well happen to build a .NET DLL for a Silicon Mac (or an ARM-based Surface laptop). I'll try and see what I can do to cover those cases and update my PR accordingly.

Reply:
@greglandrum For the ARM matter, I have tried to put in something sensible, with the caveat that it is untested on an actual ARM CPU, but it least provides a framework that someone who actually attempts this in the future and fails can hopefully tweak.

Reply:
That looks fine to me. When I have some time available, I will try and get an C# build env set up on my M1-based Mac and see if it works, but we can merge now.